### ISSUE TITLE
NoLimits 2 Roller Coaster Simulation (301320)

### ISSUE THREAD

**Issue by HonkingGoose** [ValveSoftware/Proton#825]
[Update 2.5.8 released](https://steamcommunity.com/games/301320/announcements/detail/3172148783711065560)
> Improved Linux/SteamOS/Wine compatibility

This needs retesting. However I no longer have a Linux computer, so I can't test this myself unfortunately.

---

# Compatibility Report
- Name of the game with compatibility issues: NoLimits 2 Roller Coaster Simulation
- Steam AppID of the game: 301320

## System Information
- GPU: GTX 780 Ti
- Driver/LLVM version: Nvidia 418.56
- Kernel version: 4.18.0-16-generic
- Link to full system information report as [gist](https://gist.github.com/HonkingGoose/99b027658c2a2d5ef99dc90aee5f97dd)
- Proton version: proton-4.2-1
- Game version: 2.5.7

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms

Game won't start on Proton 4.2-1 (it didn't work on earlier versions of Proton either!).
However the error message the game gives is different now compared to earlier Proton versions.

## Reproduction

1. Start game from Steam.
1. Game doesn't reach game-menu or splash screens.
1. Game immediately gives a Fatal Error message: ```The application failed with: Required files are missing. Make sure this software was installed correctly. The application will quit now.```
1. Game stops, and returns to Steam launcher.

Now to be clear: I've uninstalled the game trough Steam, and deleted the No Limits 2 folder that remained. And reinstalled the game from scratch. The game still throws the same error.

## Proton log:
[steam-301320_proton_4.2.1.log](https://github.com/ValveSoftware/Proton/files/3010927/steam-301320_proton_4.2.1.log)

**Comment by HonkingGoose** [comment]
I've also tested on Ubuntu 18.04, with the latest Nvidia driver (396.54) on the same computer. Same problems.

The Ubuntu 18.04 configuration can be found here:
[gist](https://gist.github.com/HonkingGoose/5b35f2af620286153a22e509f9b5e82a)

**Comment by HonkingGoose** [comment]
Re-tested on Proton 3.7-5 Beta, same problem, same error messages regarding D3D11 renderer.

**Comment by HonkingGoose** [comment]
Re-tested on Proton 3.7-6 Beta, same problem, with the same error messages regarding D3D11 renderer.

**Comment by doitsujin** [comment]
Hello,

the game requires native `d3dcompiler_43` DLLs for now. With that, I got the Demo to run, albeit it is a bit unstable - enabling fullscreen mode hangs the game, and the game seems to have some input-related issues, moving the mouse can cause hangs as well.

**Comment by HonkingGoose** [comment]
Hi @kisak-valve 

I see that you've marked my previous comments as outdated (which they are). This leaves me with some questions though...

What is the preferred way of updating findings whenever a new Proton release comes out?

* Should the updated report go in a **new comment**, below the older ones?
* Or should I **update a older comment** with my new findings?

I ask this, so that I don't create unnecessary busywork for you, you're busy enough as it is. :smile: 

On a related note, it might be helpful if some of the unwritten rules of the issue tracker are made explicit in a FAQ/Wiki page or something similar. 
It's hard for me to know if I'm doing right by you, because nothing is written down on the preferred way to do things. I imagine this guidance will also be helpful to the people who have never used GitHub before, and aren't familiar with the way things run here.
Making things explicit might also reduce your workload as moderator, so that you don't have to clean up things afterwards.

I don't know how else to contact you, or I would've sent a direct message/email for this, but it seems Github doesn't allow for this.

Greetings, 

HonkingGoose

**Comment by kisak-valve** [comment]
Hello @HonkingGoose, in general, this issue tracker should follow the common [conduct](https://github.com/ValveSoftware/steam-for-linux#conduct) section found in other Valve issue trackers.

Repeated retesting and posting refreshed logs does not add much value unless you find something has changed between releases.

How that code of conduct could be expressed so that it would be clear for the issue tracker and made separate from the code base is a separate topic for a Proton dev and I'm not interested in getting tangled in the needless drama surrounding codes of conduct.

It also should be made clear that I am not a Proton dev, rather, I'm a moderator helping keep the issue trackers usable for those who are working on the projects.

Discussion moved to #1664.

**Comment by HonkingGoose** [comment]
Hello @kisak-valve 

## Updating reports:
Thanks for the link to the code of conduct. It does indeed mention:
>Do not repeatedly update an open issue remarking that the issue persists.

You also say it adds no value, so I'll stop needlessly updating things.

## "Housekeeping" rules:

I think I wasn't clear enough, I'm not thinking of a "code of conduct" but more of something like this; some basic "housekeeping" rules:
1.  **One** issue per unofficially supported game.
1. Separate issues for each bug for officially supported/whitelisted games.
1. Title of issue should contain: [official name of game from steam store] + appid number + short description of issue.
1. Don't add your own re-tests/refreshed logs to a issue report, unless things change.
1. Do add references to other relevant issues. 

I've seen some of those things listed trip people up, and I also see you fixing things afterwards, so that's why I suggested some basic guidelines, so that your work might be a bit easier.

## Should I open a issue? It's your choice on this...

Should I open a new issue suggesting "house rules" like this to be added on a FAQ/Wiki somewhere?
Or would that just lead to trolls/harassment/drama?

I'll leave that decision with you, because you would have to moderate the discussion in that issue. So whatever you say, goes. I have no intention of getting involved in drama. 

I understand that you're the moderator and no developer. :+1: 

## TLDR:
* I'll stop refreshing issues needlessly. 
* You decide if there should be a issue asking for "housekeeping rules".

**Comment by kisak-valve** [comment]
@HonkingGoose, yes, please take this to a separate issue report and clean things up here so that this issue stays on topic (301320).

**Comment by reissecup** [comment]
me too

**Comment by HonkingGoose** [comment]
The patch notes for No Limits 2 version `2.6.3.2` say: [^quote]

> - Fixes a few problems including a crash when running on Linux with Wine/Proton.

So you all might want to re-test the game and see if it runs out-of-the-box now.

[^quote]: https://store.steampowered.com/news/app/301320/view/3682291055857373618?l=english

**Comment by playerlats9712** [comment]
Try using Proton 9 and setting Launch Arguments, the Software works for me without problems.

---

## Hardware
Ryzen 9 5950X
RTX 4060

---

## Software
OS: Ubuntu 26.04 LTS
Proton Version: 9.0-4
Launch Arguments: `PROTON_USE_WINED3D=1 %command%`

---

## Steam and Game Info
Steam Version: 1779918128
NoLimits 2 (2.6.8.1 Windows x86_64)
Renderer: OpenGL 4.4
Graphics Device: NVIDIA GeForce RTX 4060/PCIe/SSE2
